.github/workflows/pypi.yml

```
@@ -38,23 +38,24 @@
+
       - name: Install dependencies
         run: |
-          pip3 install wheel packaging==23.2
-          pip3 install --no-build-isolation -e .
-          pip3 install -r requirements-dev.txt -r requirements-tests.txt
+          uv pip install --system wheel packaging==23.2
+          uv pip install --system --no-build-isolation -e .[dev]
 
       - name: Extract tag name
         id: tag
         run: echo ::set-output name=TAG_NAME::$(echo $GITHUB_REF | cut -d / -f 3)
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)